src/stories/ChartExamples.story.tsx

```diff
@@ -142,56 +143,71 @@ const UserGrowthAreaStory: StoryFn<typeof Chart> = (args): ReactElement => {
 			<Axis position="bottom" baseline />
 			<Axis position="left" grid title="Users" />
 			<Area dimension="x" metric="y" order="order" scaleType="point">
-				<ChartTooltip>{dialogCallback}</ChartTooltip>
-				<ChartPopover width={200}>{dialogCallback}</ChartPopover>
+				<ChartTooltip>{generateCallback('tooltip')}</ChartTooltip>
+				<ChartPopover width={200}>{generateCallback('popover')}</ChartPopover>
 			</Area>
 			<Legend highlight descriptions={userGrowthDescriptions} />
 		</Chart>
 	);
 };
 
-const dialogCallback = (datum: Datum, close?: () => void) => {
-	return (
-		<Content UNSAFE_className="userGrowth-dialog">
-			<Flex direction="column">
-				<Flex direction="row" justifyContent="space-between">
-					<div>
-						<div>{datum.x}</div>
-						<div>{datum.series}</div>
-						<div>{Math.abs(datum.y as number).toLocaleString()} users</div>
-					</div>
+/** Generates identical return callbacks but each has a custom Storybook Action Name for a better dev experience. */
+const generateCallback = (variant: 'popover' | 'tooltip') => {
+	const actionName = {
+		popover: 'ChartPopover',
+		tooltip: 'ChartTooltip',
+	};
+
+	const callback = (datum: Datum, close?: () => void) => {
+		action(`${actionName[variant]}:callback`)(datum);
```

Comment:
Now the Storybook console prints which callback triggered what element 

<img width="865" alt="Screenshot 2024-04-01 at 10 37 39 AM" src="https://github.com/adobe/react-spectrum-charts/assets/52969178/bb03f0d7-8200-4148-b351-6e920f51c5ee">